src/test/java/org/apache/mesos/testutils/OfferRequirementTestUtils.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.mesos.testutils;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.offer.InvalidRequirementException;
+import org.apache.mesos.offer.OfferRequirement;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * This class provides utility methods for tests concerned with OfferRequirements.
+ */
+public class OfferRequirementTestUtils {
+    public static OfferRequirement getOfferRequirement(Protos.Resource resource) throws InvalidRequirementException {
+        return getOfferRequirement(Arrays.asList(resource));
```

Comment:
What does this abstraction buy us? I understand the desire to keep things dry, but sometimes adding an extra layer of indirection without much logic can hamper readability, and I like the idea of tests being super-explicit since they often function as a de facto spec.